shipyard/src/main/java/org/jclouds/shipyard/features/ContainersApi.java

```diff
@@ -64,6 +66,7 @@
    void restartContainer(@PathParam("id") String id);
    
    @Named("containers:deploy")
+   @Fallback(NullOnContainerResourceUnavailableAnd500.class)
```

Comment:
Exactly as I have it. That is literally the content from the jclouds-wire.log I posted.